### DISCUSSION TITLE
Event that fires on asset overwrite ?

### DISCUSSION THREAD

**Discussion by alexmf23** [pimcore/pimcore#14168] [Q&A / Support]
Hello, 
is there an event that fires when overwriting asset image ?
![Screenshot_20230129_111010](https://user-images.githubusercontent.com/63473281/215358775-f0b32e0a-dcd9-470f-a1e5-af5030488a16.png)

**Comment by mattamon** [comment]
Hey @alexmf23,

Unfortunately there is no event that fires when an asset is overridden. 

https://github.com/pimcore/pimcore/blob/10.5/lib/Event/AssetEvents.php
Here are all events so far but this could be a good addition. 

Can you create a PR for that?

**Reply by frithjof** [reply]
i guess update events should trigger on overwrite
